### ISSUE TITLE
Not user friendly

### ISSUE THREAD

**Issue by rscipher001** [Alok255/Text-To-Speech-in-Android-Example-#1]
There are many useless things found in this project.

1. Nothing is described in proper way.
2. Project is compress in zip file so I can't see what's inside may be you can distribute **virus inside zip** (Who knows).
3. After downloading I found lot of useless files are included. Those files are automatically generated by Android Studio hence can be removed from project to reduce size.
4. A DC super-girl photo found inside, Useless 1.2 MB data in project.

In short this project is 

> **just a show off**

. Alok255 don't care about the audience. Dear Alok255 if you will solve these issue I will replace my comment with good rating comments
